core/lib/generators/spree/install/install_generator.rb

```diff
@@ -149,23 +151,26 @@ def load_sample_data
     end
 
     def install_routes
-      insert_into_file File.join('config', 'routes.rb'), after: "Rails.application.routes.draw do\n" do
-        <<-ROUTES
+      routes_file_path = File.join('config', 'routes.rb')
+      unless File.read(routes_file_path).match Regexp.new(SPREE_MOUNT_ROUTE)
```

Comment:
Treating `SPREE_MOUNT_ROUTE` as a regex is going to work fine because it doesn't have any regex characters (`*`, `(`, `.`, etc), but it feels like the wrong thing to do.

This should probably be

    File.read(routes_file_path).include?(SPREE_MOUNT_ROUTE)

Reply:
Check